designer/client/lists-edit.js

```diff
@@ -12,27 +12,29 @@ class ListsEdit extends React.Component {
     })
   }
 
-  onClickAddList = (e, list) => {
+  onClickAddList = async e => {
     e.preventDefault()
-
+    const { data } = this.props
+    const id = await data.getId()
     this.setState({
+      id: id,
       showAddList: true
     })
   }
 
   render () {
     const { data } = this.props
     const { lists } = data
-    const list = this.state.list
+    const { list, id } = this.state
 
     return (
       <div className='govuk-body'>
         {!list ? (
           <div>
             {this.state.showAddList ? (
-              <ListEdit list={{}} data={data}
-                onEdit={e => this.setState({ showAddList: false })}
-                onCancel={e => this.setState({ showAddList: false })} />
+              <ListEdit list={{}} data={data} id={id}
```

Comment:
can you get away with not providing ListEdit the list property? since it's an empty object anyway

Reply:
Fixed